### PR TITLE
x11-drivers/xf86-input-mtrack: Fix x11 proto deps

### PR DESCRIPTION
**Pull request by simonvanderveldt**
Should fix bug 615384 https://bugs.gentoo.org/show_bug.cgi?id=615384

### CODE REVIEW

File: x11-drivers/xf86-input-mtrack/xf86-input-mtrack-0.4.1-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+SRC_URI="https://github.com/p2rkw/xf86-input-mtrack/tarball/v${PV/_/-} -> ${P}.tar.gz"
+IUSE="debug"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="amd64 ~arm x86"
+
+RDEPEND="${RDEPEND}
+	>=sys-libs/mtdev-1.0"
+DEPEND="${DEPEND}
```
Comment (SoapGentoo) [inline x11-drivers/xf86-input-mtrack/xf86-input-mtrack-0.4.1-r1.ebuild]: what's the point of reincluding the same `DEPEND`/`RDEPEND` again?

File: x11-drivers/xf86-input-mtrack/xf86-input-mtrack-0.4.1-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	>=sys-libs/mtdev-1.0
+	x11-proto/inputproto
+	x11-proto/randrproto
+	x11-proto/videoproto
+	x11-proto/xineramaproto
+	x11-proto/xproto"
+
+DOCS=( "README.md" )
```
Comment (SoapGentoo) [inline x11-drivers/xf86-input-mtrack/xf86-input-mtrack-0.4.1-r1.ebuild]: you sure you need this? `einstalldocs` should catch this

**Comment by simonvanderveldt** [comment]
@SoapGentoo I've not touched the rest of the ebuild expect for adding the deps. Should I rename the old ebuild to make that clearer?
Also, do you want to fix the issues you mentioned in this PR or should that be a separate thing?

**Comment by SoapGentoo** [comment]
@simonvanderveldt Fix them in an additional commit

**Comment by simonvanderveldt** [comment]
@SoapGentoo Should be fixed now

**Comment by simonvanderveldt** [comment]
@SoapGentoo Is this PR OK with the additional changes I added?

**Comment by simonvanderveldt** [comment]
Closing because of no reponse

**Comment by monsieurp** [comment]
Could you please file another PR regarding this bug? I will have a look at it.

**Comment by monsieurp** [comment]
@simonvanderveldt

**Pull request closed without merging**